File: PASCAL/Cap05/c05ex04.pas
program c05ex04;

var
  RESP: String;
  N, R: Integer;

begin
  RESP := 'S';
  while not (UpCase(RESP) <> 'S') do
    begin
      Write('Entre um valor numerico: ');
      ReadLn(N);

      R := N * 3;

      WriteLn('Resultado = ', R);
      WriteLn;

      Write('Deseja continuar? (S/N): ');
      ReadLn(RESP);
      WriteLn;
    end;

  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: CSHARP/Cap05/c05ex04.cs
// c05ex04.cs

using System;
using static System.Console;

class Program
{
    static void Main(string[] args)
    {
        char RESP;
        int N, R;

        RESP = 'S';
        while (!(char.ToUpper(RESP) != 'S'))
        {
            Write("Entre um valor numerico: ");
            N = int.Parse(ReadLine());

            R = N * 3;

            WriteLine($"Resultado = {R}\n");

            Write("Deseja continuar? (S/N): ");
            RESP = char.ToUpper(ReadLine()[0]);

            WriteLine();
        }

        Write("\nTecle <Enter> para encerrar... ");
        ReadLine();
    }
}
